taxi/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+from taxi.validators import validate_license_number
+
+
+class DriverCreateForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "email",
+            "license_number",
+        )
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+        return validate_license_number(license_number)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data.get("license_number")
+        return validate_license_number(license_number)
+
+
+class CarCreateForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The `drivers` field in `CarCreateForm` is using `get_user_model().objects.all()` for the queryset. This should be filtered to include only `Driver` instances, as the form is likely intended to select drivers, not all users. Consider using `Driver.objects.all()` instead.

Reply:
done